setup.py

```diff
@@ -240,6 +241,7 @@
     ],
     "tensorflow_gpu": ["tensorflow-gpu>=2.2.0,!=2.6.0,!=2.6.1"],
     "torch": ["torch"],
+    "polars": ["polars"],
```

Comment:
no need to add it here
```suggestion
    "polars": ["polars"],
```

Reply:
Noted